pyaedt/maxwell.py

```diff
@@ -1045,3 +1045,52 @@ def assign_vector_potential(self, input_edge, vectorvalue=0, bound_name=None):
             self.boundaries.append(bound)
             return bound
         return False
+
+    @aedt_exception_handler
+    def assign_master_slave(self, master_edge, slave_edge, reverse_master=False, reverse_slave=False,
+                            same_as_master=True, bound_name=None):
+        """Assign Master/Slave Boundary Conditions to 2 edges of the same object.
```

Comment:
```suggestion
        """Assign master and slave boundary conditions to two edges of the same object.
```

Reply:
@akaszynski At one point, Ansys had an initiative to remove terminology such as "master" and "slave". Should something be done here?

Reply:
@PipKat that a good point. However, I don't think we can rename anything in this case, simply because this is still the terminology that AEDT continues to use.